components/CollectionCard.js

```diff
@@ -179,9 +187,38 @@ export default function CollectionCard({
         <StyledLink href={`/collections/${collection._id}/archive`}>
           Archive
         </StyledLink>
-        <DeleteButton onClick={toggleConfirmation} type="button">
-          delete
-        </DeleteButton>
+
+        <Modal
+          isOpen={isModalOpen}
+          onClose={() => setIsModalOpen(false)}
+          title="Update Colelction"
+          needsPortal={true}
+        >
+          <CollectioncardForm
+            onSubmit={handleSubmit}
+            collections={collections}
+            initialValues={{
+              _id: collection._id,
+
+              collectionId: collection?.collectionId || "",
+              title: collection?.title,
+            }}
```

Comment:
You will need only the `_id` and `title` here as `initialValues`.

```suggestion
            initialValues={{
              _id: collection._id,
              title: collection?.title,
            }}
```